.github/workflows/performance.yml

```
@@ -53,7 +46,7 @@ jobs:
   # - Ensures version-controlled files are not modified or deleted.
   performance:
     runs-on: ubuntu-latest
-    if: ${{ github.repository == '10up/wordpress-develop' || github.event_name == 'push' }}
+    if: ${{ github.repository == 'WordPress/wordpress-develop' }}
```

I think we want this to run on PRs too, but we'll only want to post to the dashboard if it's a push to event (not on pull requests).

```suggestion
    if: ${{ github.repository == 'WordPress/wordpress-develop' || github.event_name == 'pull_request' }}
```

---

Thanks Joe for review. Updated in https://github.com/10up/wordpress-develop/pull/41/commits/e66687a08628d2348048dba0ec6cddc19a6d2937